brushfire-core/src/main/scala/com/stripe/brushfire/Tree.scala

```diff
@@ -26,14 +36,15 @@ object Tree {
         val edges = split.predicates.toList.map {
           case (pred, _) =>
             val newInstances = instances.filter { inst => pred(inst.features.get(splitFeature)) }
+            val annotation = Semigroup.sumOption(instances.map { inst => annotator.create(inst.metadata) })
```

Comment:
Is there a non-empty list somewhere in algebird with a monoid instance? .. or should I just use `.get`?